Add Basic baremetal application example

kegilbert opened the pull request:
Add example page for a starting bare-metal application the baremetal book.

@bulislaw

kegilbert committed:
Add Basic baremetal application example

Commit:
Edit baremetal_example.md

Edit file, mostly for precise language and example formatting.

iriark01 commented on docs/baremetal_example.md:
```diff
@@ -0,0 +1,5 @@
+## Bare metal example
+
+To start using the bare metal profile on Mbed OS 5, please see the blinky bare metal example:
```
Isn't it normally Blinky?

AnotherButler replied:
We have a new example that's basically blinky for bare metal; it's different from the standard blinky.

AnotherButler replied:
Oh, do you just mean capitalization?

iriark01 replied:
Yes, capitalisation.

SenRamakri commented on docs/baremetal_example.md:
```diff
@@ -0,0 +1,5 @@
+## Bare metal example
+
+To start using the bare metal profile on Mbed OS 5, please see the blinky bare metal example:
```
How about changing this sentence to - "To learn about using the bare metal profile on Mbed OS 5,... ". 

Everything else looks great.

Commit:
Update baremetal_example.md

Fix phrasing as suggested in comments, and capitalize "Blinky" for consistency as suggested in comments.

AnotherButler commented:
ping @bulislaw
